vocode/streaming/utils/create_task.py

```diff
@@ -6,25 +6,12 @@
 tasks_registry = []
 
 
-def log_if_exception(reraise_cancelled: bool, future: asyncio.Future) -> None:
-    try:
-        if exc := future.exception():
-            logger.exception(
-                f"Vocode wrapped logger; exception raised by task {future}: {exc}",
-                exc_info=exc,
-            )
-    except asyncio.CancelledError:
-        if reraise_cancelled:
-            raise
-
-
-def asyncio_create_task_with_done_error_log(
+def asyncio_create_task(
     *args,
     reraise_cancelled: bool = False,
```

Comment:
this would be a regression to remove this, right? remind me again what happens to tasks normally when they get cancelled

Reply:
I'm not removing it though? It's just a method rename to not have "with done error log"

Reply:
Are you referring to the reraise_cancelled part?

Reply:
yes, I'm referring to the reraise_cancelled part

Reply:
reading back the code, this is safe - all `reraise_cancelled = True` does is make more visible a cancel error when it actually gets cancelled